project/AppDependencies.scala

```diff
@@ -7,7 +7,7 @@ object AppDependencies {
 
   val compile = Seq(
     "uk.gov.hmrc"             %% "bootstrap-frontend-play-28" % "5.17.0",
-    "uk.gov.hmrc"             %% "play-frontend-hmrc"         % "1.31.0-play-28",
+    "uk.gov.hmrc"             %% "play-frontend-hmrc"         % "3.4.0-play-28",
```

Comment:
this seems like a big jump in versions, do we need to do this across all our apps?

Reply:
Only a few versions after the one we used to use and it keeps the repo nice and up-to-date.